assets/ops/setup_security.sh

```diff
@@ -6,11 +6,4 @@ fi
 
 echo "Adding QL Certificate as Trusted..."
 
-CERT_FILE=Quicken_Loans_Root_CA.crt
-CERT=https://git.rockfin.com/raw/SKluck/docker-images/master/.shared/certificates/$CERT_FILE
-CERT_DIR=/etc/ssl/certs
-
-curl -sSl -o $CERT_DIR/$CERT_FILE $CERT
-
-echo cacert=/etc/ssl/certs/Quicken_Loans_Root_CA.crt > ~/.curlrc
-
+curl -sLk $QL_CERTS | bash -
```

Comment:
Maybe we can pass the cert URL as an argument on the command line (vs adding it to .env as I believe you're proposing)? I would really prefer to have such a thing directly in the repo... is there someone in particular I can talk to about this?

Reply:
keeping in mind the idea to keep it simple. I don't see the advantage of having the user run another command during setup. This would just be a copy & paste of `.env`, which was designed for this purpose, and follows the conventions used in other projects. Also, the `.env` file will again be needed in the future when we implement authentication.

Reply:
continuing with my thoughts around removing anything referencing QL, the environment vars should be renamed to something more generic. Maybe the command itself reworked to be more generic overall?